### PR TITLE
Refactor/move logic to pcms

### PR DESCRIPTION
**Pull request by joacimastrom**
# Jira Issue: []

## What?
- Move formatting of perils and icons to PCMS
- Cleanup contract endpoint
- Put contract endpoint under /public as the other ones

## Why?
- Cleanup to make PCMS and giraffe more maintainable

## Optional checklist
- [ ] Unit tests written
- [X] Tested locally
- [X] Codescouted

### CODE REVIEW

**Review by cpiehl1 — APPROVED** [review]
🚀

File: src/api/upstreams/promiseCms.ts [hunk 1]
```diff
@@ -75,62 +47,32 @@ export interface ContractInfo {
...
-}
-
 export const createPromiseCmsClient = (
   client: HttpClient,
 ): PromiseCmsClient => ({
   getContractInfo: async ({ contractType, locale, date }) => {
     try {
       const res = await client.get(
-        `/contracts/${contractType}/${locale}${date ? `?date=${date}` : ''}`,
+        `/public/contract/${contractType}/${locale}${
```
Comment (cpiehl1) [inline src/api/upstreams/promiseCms.ts:56]: Normally plural `contracts` would be used, right?

Comment (joacimastrom) [reply src/api/upstreams/promiseCms.ts:56]: Yeah I think you're right, got them mixed up

**Review by FredrikAreschoug — APPROVED** [review]
:ship:

**Pull request merged**